File: result/translate/transcoder/python2java/java/MINIMUM_PERIMETER_N_BLOCKS.java
public static int f_gold ( int n ) {
  int l = ( int ) Math . sqrt ( n ) ;
  int sq = l * l ;
  if ( ( sq == n ) && ( sq == n ) ) {
    return l * 4 ;
  }
  else {
    int row = ( int ) ( n / l ) ;
    int perimeter = 2 * ( l + row ) ;
    if ( ( n % l != 0 ) && ( n % l != 0 ) ) {
      perimeter += 2 ;
    }
    return perimeter ;
  }
}

File: result/translate/chatgpt/java2python/python/MINIMUM_PERIMETER_N_BLOCKS.py
def f_gold(n):
    l = int(n ** 0.5)
    sq = l * l
    
    if sq == n:
        return l * 4
    else:
        row = n // l
        perimeter = 2 * (l + row)
        
        if n % l != 0:
            perimeter += 2
        
        return perimeter